src/scan_nut.cc

```diff
@@ -133,6 +133,29 @@ s_nut_dumpdata_to_fty_message (fty_proto_t *fmsg, std::map <std::string, std::st
         }
     }
 }
+bool
+s_valid_dumpdata (std::map <std::string, std::string> &dump)
+{
+  if(dump.find("device.type") == dump.end()) {
+    zsys_error("No subtype for this device");
+    return false;
+  }
+
+  if(dump.find("device.model") == dump.end() && dump.find("device.ups.model") == dump.end() &&
```

Comment:
either device.model (new) or ups.model (legacy) but no combo.
that just applies to subdevices (non master of daisychain) so device.x.ups.model, which is also served as device.x.model